### PR TITLE
Pulse animation

### PR DESCRIPTION
**Pull request by Protectator**
Beaucoup de params, très modulable. 😄 

https://user-images.githubusercontent.com/1529017/204115744-5da9cc21-21c4-49b4-b192-f1c93ec7ffe0.mp4

### CODE REVIEW

File: server/lamp/pulse.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          ], default: "rl" },
+        { name: "Speed", type: "number", default: 10, min: 1, max: 100, step: 1, display: 'range' },
+        { name: "# of Pulses", type: "number", default: 1, min: 1, max: 10, step: 1, display: 'range' },
+        { name: "Rotation speed", type: "number", default: 0, min: 0, max: 360, step: 1, display: 'range' },
+        { name: "Top Led Number", type: "number", default: TOP_LED_NB, min: 0, max: NB_LED, step: 1, display: 'range' }
+    ];
+
+    public animate(t, display, options) {
+        const [color1, color2, branches, speed, pulses, rotationSpeed, topLedNb] = options;
+        const realSpeed = speed / 10;
```
Comment (BinaryBrain) [inline server/lamp/pulse.ts:26]: This should change proportionally with LAMP_FPS

File: server/lamp/pulse.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            {optionValue: "r", label: "Counter-Clockwise"},
+          ], default: "rl" },
+        { name: "Speed", type: "number", default: 10, min: 1, max: 100, step: 1, display: 'range' },
+        { name: "# of Pulses", type: "number", default: 1, min: 1, max: 10, step: 1, display: 'range' },
+        { name: "Rotation speed", type: "number", default: 0, min: 0, max: 360, step: 1, display: 'range' },
+        { name: "Top Led Number", type: "number", default: TOP_LED_NB, min: 0, max: NB_LED, step: 1, display: 'range' }
+    ];
+
+    public animate(t, display, options) {
+        const [color1, color2, branches, speed, pulses, rotationSpeed, topLedNb] = options;
```
Comment (BinaryBrain) [inline server/lamp/pulse.ts:25]: Colors should be stored on dots (if possible) to be compatible with Random Colors and future features.

**Review by BinaryBrain — CHANGES_REQUESTED** [review]

File: server/lamp/pulse.ts (hunk 2 above)
Comment (Protectator) [reply server/lamp/pulse.ts:25]: Voilà : https://github.com/Pecamo/orbital/issues/77

File: server/lamp/pulse.ts (hunk 1 above)
Comment (Protectator) [reply server/lamp/pulse.ts]: Done

**Review by BinaryBrain — APPROVED** [review]

**Pull request merged**